documentation/architecture/code.md

```diff
@@ -0,0 +1,18 @@
+
+# Classes and relations
+The figures here were obtained using [SourceTrail](https://github.com/CoatiSoftware/Sourcetrail). The
```

Comment:
What do the different colours in the figures mean?

Reply:
will add comment